Automattic/wp-calypso
# Jetpack Sync: Try sync status UI

**Note:** This PR is meant as a proof of concept. It will be broken up into smaller PRs where tests and finishing touches are addressed.

We will be working on moving the Jetpack Sync UI into Calypso, and as part of that, we need to be able to store and get data.

We are currently working on the endpoint, and haven't fully fleshed out what it looks like. But, I feel like this PR gives us a head start for when we finish the endpoint in the next couple of days.

To test:

- Checkout `add/jetpack-sync-ui` branch
- Checkout latest Jetpack `master`
- Go to `/settings/general/$site`
- Click "Perform full sync" button
- Go to `$site/wp-admin/admin.php?page=jetpack-sync` and ensure sync is running


Test live: https://calypso.live/?branch=add/jetpack-sync-ui
